docs/src/snap/reference/certificates.md

```diff
@@ -0,0 +1,45 @@
+# Cluster Certificates
+
+## **CertificateAuthorities (CA)**
+
+| **Common Name**              | **Usage**                            | **Path on Disk**                         | **Used For**                                |
```

Comment:
```suggestion
| **Certificate Authority (CA) Name**              | **Purpose**                            | **File Location**                         | **Primary Function**                                |
```
Not sure if I broke your table format please check :)

Reply:
`Common name` is an official RFC for the certificate field
https://support.dnsimple.com/articles/what-is-common-name/

Reply:
Learnt something new :)